src/main/java/gui/misc/TweakingHelper.java

```diff
@@ -8,18 +8,19 @@
 public final class TweakingHelper {
 
     // four main colors used throughout application. Experiment a little!
```

Comment:
I'll replace these with getters and setters later, if we ever add an options menu or something.